portfolio.js

```diff
@@ -32,3 +37,90 @@ for (let i = 0; i < listItem.length; i += 1) {
+  const portfolioHeading = card[indexValue].children[1].children[0];
+  const canopyItem = card[indexValue].children[1].children[1];
+  const cardImage = card[indexValue].children[0];
+  const technology = card[indexValue].children[1].children[3];
+
+  const project = [{
+    closePopup: './images/close2.png',
+    portfolioHeading: `${portfolioHeading.innerHTML}`,
+    projectImage: `${cardImage.innerHTML}`,
+    canopyItem: `${canopyItem.innerHTML}`,
+    cardImage: `${cardImage.innerHTML}`,
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry'
+    + 'standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type'
+    + 'specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    seeLive: './images/seeLive.png',
+    seeSource: './images/seeSource.png',
+    liveProjectLink: 'https://amrendrakind.github.io/MyPortfolio_MV/',
+    SourceCodeLink: 'https://github.com/amrendrakind/MyPortfolio_MV',
+  }];
+
+  const htmlMarkup = `
+  <div class = "popup-heading-wrap">
+    <div class="popup-heading">    ${project[0].portfolioHeading}</div>
+    <img class="close-icon" src=${project[0].closePopup}>
+  </div>
+  <div class="popup-canopy" >${project[0].canopyItem}</div>
+  <picture class="popup-cardimage">  ${project[0].cardImage} </picture>
+  <div class = "desc-tech-btn">
+    <div class = "description" >${project[0].description}</div>
+    <div class = "tech-btn">
+      <div class="popup-technology" >${technology.innerHTML}</div>
+      <div class="popup-button">
+        <button class="popup-button-liveSource">
+          See Live
+          <img src=${project[0].seeLive} alt="">
+        </button>
+        <button class="popup-button-liveSource">
+          See Source
+          <img src=${project[0].seeSource} alt="">
+        </button>
+      </div>
+    </div>
+  </div>
+  `;
+
+  section.innerHTML = htmlMarkup;
+
+  const default1 = document.querySelector('.default1');
+  const mediaQuery = window.matchMedia('(min-width: 992px)');
+  if (mediaQuery.matches) {
+    const popupImage = default1.querySelector('.popup-image');
+    popupImage.style.maxHeight = '586px';
+    popupImage.style.width = '100%';
+  }
+  const blurPage = document.querySelectorAll('.blur');
+
+  for (let i = 0; i < blurPage.length; i += 1) {
+    blurPage[i].style.filter = 'blur(10px)';
+  }
+  const scrollStop = document.querySelector('body');
+  scrollStop.style.overflow = 'hidden';
+  const closeIcon = document.querySelector('.close-icon');
+  section.classList.toggle('popup');
+  function display() {
+    section.classList.toggle('popup');
+    scrollStop.style.overflow = 'scroll';
+    for (let i = 0; i < blurPage.length; i += 1) {
+      blurPage[i].style.filter = 'blur(0)';
+    }
+  }
+  closeIcon.addEventListener('click', display);
```

Comment:
- [ ] Good job creating this section dynamically with javascript 👏. Kindly note that there should be changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript as you did here 💪🏽.

Reply:
I am not able to understand. Can you make me understand via zoom call?

Reply:
Hi @amrendrakind 👋🏽,
I can't join you on zoom at the moment as I'm trying to finish a task here 🙏🏽, but what I mean is after creating your code dynamically with javascript, existing code should be removed from your HTML work section. Do you get it?💪🏽

Reply:
@rotimiazeez, Do you mean that code  (https://github.com/amrendrakind/MyPortfolio_MV/blob/popup/portfolio.js#L100) I inserted in the newly created section should be removed dynamically after use is done with a click of the close button?

Reply:
@rotimiazeez Please respond?

Reply:
@rotimiazeez, Hey can you respond to my query?

Reply:
I mean since you already created this with javascript, then the details in your index.html should removed.
For example
![screenshot (27)](https://user-images.githubusercontent.com/44624138/157769391-ffa0185b-eadf-41fa-af98-8f6aee2c1809.png)
this part and the like should be removed and only created with the dynamic javascript when the page loads.😉

Reply:
@rotimiazeez, I am still not clear. I made some changes and pushed. It will be great if you can check and confirm.

Reply:
https://microverse.zoom.us/j/8703243084?pwd=TENlbHJUeDgxWUxPWE95UXd6dCtTUT09 join me here for a bit

Reply:
Kindly join using the above link so I can explain what you need to do before I continue what I'm doing 🙏🏽

Reply:
Give me 2 min. I am joining

Reply:
@rotimiazeez I implemented changes. Can you guide me on whether it is good enough or I need to make more changes?